src/main/java/mate/academy/hibernate/relations/exception/ActorNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package mate.academy.hibernate.relations.exception;
+
+public class ActorNotFoundException extends RuntimeException {
+    public ActorNotFoundException(String message) {
+        super(message);
+    }
+}
```

Comment:
we shouldn't write new exception for every class
if you wish, you can add `EntityNotFoundException`